README.md

```diff
@@ -1,10 +1,10 @@
-# renga-storage
-Renga Storage Service
+# renku-storage
+Renku Storage Service
 
-**Renga is currently undergoing a major restructuring effort. For a preview, you can look
+**Renku is currently undergoing a major restructuring effort. For a preview, you can look
 at the development branch, but keep in mind it is highly volatile.**
 
-Documentation: https://renga.readthedocs.io/en/latest/developer/services/storage_service.html
+Documentation: https://renku.readthedocs.io/en/latest/developer/services/storage_service.html
```

Comment:
keep old?

Reply:
we can for now, but a new description will be needed in the docs

Reply:
renku docs build now so this change is good to go